lms/static/js/student_account/views/AccessView.js

```diff
@@ -161,7 +162,8 @@
                         fields: data.fields,
                         model: model,
                         thirdPartyAuth: this.thirdPartyAuth,
-                        platformName: this.platformName
+                        platformName: this.platformName,
+                        registrationFooter: this.registrationFooter,
```

Comment:
I would move our new line to 163 or 164, and remove the added comma on the platformName one